scripts/run_frontend_tests.py

```diff
@@ -152,10 +162,32 @@ def main(args: Optional[Sequence[str]] = None) -> None:
             # the line to print it.
             print(line.decode('utf-8'), end='')
             output_lines.append(line)
+        # Download the combined-tests.js file from the web-server.
+        if ('Executed' in line.decode('utf-8') and
+            not combined_test_spec_downloaded and
+            parsed_args.download_combined_frontend_file):
+            download_task = subprocess.Popen(
+                ['wget',
+                'http://localhost:9876/base/core/templates/' +
+                'combined-tests.spec.js',
+                '-P',
+                os.path.join('../karma_coverage_reports')])
+            combined_test_spec_downloaded = True
     # Standard output is in bytes, we need to decode the line to print it.
     concatenated_output = ''.join(
         line.decode('utf-8') for line in output_lines)
-
+    if download_task:
+        # Wait for the wget command to download the combined-tests.spec.js
+        # file to complete.
+        download_task.wait()
```

Comment:
Is there any issue with calling wait just after running the wget command?

Reply:
No, moved the line.